apps/server/src/modules/tool/controller/dto/response/lti11-tool-config.response.ts

```diff
@@ -0,0 +1,35 @@
+import { LtiMessageType } from '@src/modules/tool/interface/lti-message-type.enum';
+import { ToolConfigType } from '@src/modules/tool/interface/tool-config-type.enum';
+import { ExternalToolConfigResponse } from '@src/modules/tool/controller/dto/response/external-tool-config.response';
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { LtiPrivacyPermission } from '@src/modules/tool/interface/lti-privacy-permission.enum';
```

Comment:
Please internel imports for example via ../interface or ./dto

Reply:
done